src/main/java/com/capgemini/wsb/fitnesstracker/training/api/dto/TrainingWithUsersDto.java

```diff
@@ -0,0 +1,15 @@
+package com.capgemini.wsb.fitnesstracker.training.api.dto;
+import com.capgemini.wsb.fitnesstracker.training.internal.ActivityType;
+import com.capgemini.wsb.fitnesstracker.user.api.User;
+
+import java.util.Date;
+
+public record TrainingWithUsersDto(
+        Long id,
+        User user,
```

Comment:
In dto we should use dto objects. Please user UserDto instead of UserDto.

Reply:
changed from user to userDto for all dto records that require users